x-pack/legacy/plugins/actions/server/builtin_action_types/server_log.ts

```diff
@@ -49,7 +50,7 @@ async function executor(execOptions: ActionTypeExecutorOptions): Promise<ActionT
   } catch (err) {
     return {
       status: 'error',
-      message: `error logging message: ${err.message}`,
+      message: `error in action ${id} logging message: ${err.message}`,
```

Comment:
This and all strings returned from API calls should be localized.

Reply:
Probably localization should be a separate PR - created a card for that - https://github.com/elastic/kibana/projects/26#card-24316443